Completion/Unix/Command/_stow

```diff
@@ -55,7 +55,13 @@ _stow(){
         stow_dir="$opt_args[-d]"
       elif (( $+opt_args[--dir] )) ; then
         stow_dir="$opt_args[--dir]"
+      elif (( $+STOW_DIR )) ; then
+        # if not provided from the command line, for the stow command, the stow
+        # directory is assumed to be the value of the "STOW_DIR" environment
+        # variable...
+        stow_dir="$STOW_DIR"
```

Comment:
I wonder if it would be worthwhile to check `[[ ${(t)STOW_DIR} == *export* ]]`, in order to prevent mysterious behaviour for users who set the variable but forget to export it.